website/content/docs/api-gateway/configuration/gatewayclassconfig.mdx

```diff
@@ -0,0 +1,196 @@
+
+## Configuration model
+
+* [`consul`](#consul): object | optional
+  * [`address`](#consul-address): string | optional
+  * [`authentication`](#consul-authentication): object | optional
+    * [`account`]([#consul-authentication-account): string | optional
+    * [`managed`](#consul-authentication-managed): bool  | optional
+    * [`method`](#consul-authentication-method): string | optional
+    * [`namespace`](#consul-authentication-namespace): string  | optional
+  * [`ports`](#consul-ports): object  | optional
+    * [`grpc`](#consul-ports-grpc): integer | optional
+    * [`http`](#consul-ports-http): integer | optional
+  * [`scheme`](#consul-scheme): string  | optional
+* [`copyAnnotations`](#copyAnnotations): object | optional
+  * [`service`](#copyAnnotations-service): array of strings | optional
+* [`deployment`](#deployment): object | optional
+  * [`defaultInstances`](#deployment-defaultinstances): integer | optional
+  * [`maxInstances`](#deployment-maxinstances): integer | optional
+  * [`minInstances`](#deployment-mininstances): integer | optional
+* [`image`](#image): object | optional
+  * [`consulAPIGateway`](#image-consulapigateway): string | optional
+  * [`envoy`](#image-envoy): string | optional
+* [`logLevel`](#loglevel): string | optional
+* [`nodeSelector`](#nodeselector): string | optional
+* [`serviceType`](#servicetype): string | optional
+* [`useHostPorts`](#usehostports): boolean | optional
+
+## Specification
+
+This topic provides details about the configuration parameters.
+
+### consul
+* Type: object
+* Required: optional
+
+### consul.address
+Specifies the address of the Consul server that the `Gateway` communicates with in the gateway pod. If unspecified, the pod attempts to use a local agent on the host where the pod is running.
+* Type: string
+* Required: optional
+* Default: local agent
+
+### consul.authentication.account
+Specifies the Kubernetes service account to use for authentication.
+* Type: string
+* Required: optional
+
+### consul.authentication.managed
+Set to `true` to enable deployments to run with managed service accounts created by the gateway controller. The `consul.authentication.account` field is ignored when this option is enabled.
+* Type: boolean
+* Required: optional
+* Default: `false`
+
+### consul.authentication.method
+Specifies the Consul auth method used for initial authentication by Consul API Gateway.
+* Type: string
+* Required: optional
+
+### consul.authentication.namespace
+Specifies the Consul namespace to use for authentication.
+* Type: string
+* Required: optional
+
+### consul.ports.grpc
+Specifies the gRPC port for Consul's xDS server.
+* Type: integer
+* Required: optional
+* Default: `8502`
+
+### consul.ports.http
+Specifies the Consul HTTP port to use for authentication.
+* Type: integer
+* Required: optional
+* Default: `8500`
+
+### consul.scheme
+Specifies the scheme to use for connecting to Consul. The supported values are `"http"` and `"https"`.
```

Comment:
```suggestion
Specifies the scheme to use for connecting to Consul.
```

Deleting it from the description would be consistent with the way the rest of the restructure is formated, but let me know your preference.